File: python/flamaster.py
def lettersCount(string): 
    letters = list(string)
    result = {}
    finalresult = []
    
    for letter in letters: 
        if letters.count(letter) > 2: 
            result[letter] = str(letters.count(letter))
        
        if letters.count(letter) <= 2: 
            result[letter] = str(letters.count(letter))
    
    
    for items in result.items(): 
        if int(items[1]) <=2: 
            finalresult.append(items[0])
        else: 
            for item in items: 
                finalresult.append(item)
    
    return ''.join(finalresult)
    


print(lettersCount('ABBCCCDDDDEEEEEFGGHIIJKKKL'))

File: c#/Flamaster.cs
using System;
using System.Linq;
using System.Collections.Generic;

namespace ConsoleApp5
{
    class Program
    {
        static string CountLetters(string letters)
        {
            HashSet<string> result = new HashSet<string>();
            foreach (char s in letters)
            {
                if (letters.Count(x => x == s) <= 2)
                {
                    result.Add(Convert.ToString(s));
                }
                
                if (letters.Count(x => x == s) > 2)
                {
                    result.Add(Convert.ToString(letters.Count(x => x == s)));
                }
            }
            return String.Join("", result.ToArray());
        }
        static void Main(string[] args)
        {
            Console.WriteLine(CountLetters("ABBCCCDDDDEEEEEFGGHIIJKKKL"));
        }
    }
}
